packages/storefront-lwc/src/modules/commerce/basketTotals/basketTotals.html

```diff
@@ -6,7 +6,7 @@
 -->
 <template>
     <commerce-total description="Shipping Cost" total={shippingCost} total-class="text-right"></commerce-total>
-    <!--<commerce-total description="Shipping Discount" description-class="shipping-discount" total={shippingDiscount} total-class="text-right shipping-discount"></commerce-total>-->
+    <commerce-total if:true={hasShippingDiscount} description="Shipping Discount" description-class="shipping-discount" total={shippingDiscount} total-class="text-right shipping-discount"></commerce-total>
```

Comment:
Gurpreet already had a placeholder for showing shipping level discounts. So uncommented that.